test/express/server-specs.js

```diff
@@ -43,11 +44,16 @@ describe('server', () => {
       app.get('/error', () => {
         throw new Error('hahaha');
       });
+      app.get('/pause', async (req, res) => {
+        res.header['content-type'] = 'text/html';
+        await B.delay(3000);
```

Comment:
I like it, do we need this to be quite so long though? this is not an e2e test after all. could we get away with 500ms or 1s?